infra/azure/README.md

```diff
@@ -109,12 +109,14 @@ Next, create a `global.tfvars` file from the following template and replace in t
 
 ```
 az_resource_group_name = "<resource_group_name>"
+acr_name               = "<azure_container_registry_name>"
 ```
 
-Run `terraform apply -var-file=global.tfvars`. To sync the kubernetes config, run
+Run `terraform apply -var-file=global.tfvars`. To sync the kubernetes config and login with docker, run
```

Comment:
Do you mean login to the Azure Container Registry?

Reply:
yes, I can clarify this.